soda-csi-plug-n-play-poc/deploy/kubernetes/ceph/pod.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: pod-with-raw-block-volume
+spec:
+  containers:
+    - name: fc-container
+      image: fedora:26
+      command: ["/bin/sh", "-c"]
+      args: ["tail -f /dev/null"]
+      volumeDevices:
+        - name: data
+          devicePath: /dev/xvda
```

Comment:
Will the RBD block device be always mounted at /dev/xvda? usually it is platform and virtualization mechanism dependent

Reply:
no, it is not compulsory, the user can select the desired path for mounting block volumes inside the pod.